src/UI/Microsoft.AspNetCore.Identity.UI.csproj

```diff
@@ -12,6 +12,10 @@
     <RazorCompileOnPublish>false</RazorCompileOnPublish>
     <EnableDefaultRazorGenerateItems>false</EnableDefaultRazorGenerateItems>
     <DisableFastUpToDateCheck>true</DisableFastUpToDateCheck>
+
+    <_EnableAllInclusiveRazorSdk>true</_EnableAllInclusiveRazorSdk>
```

Comment:
Needed in RCLs. This will get fixed eventually - https://github.com/aspnet/Razor/issues/2676